lib/ambientlight/index.js

```diff
@@ -56,7 +60,7 @@ AmbientLight.prototype.update = function update() {
         return self.onerror(data);
       } else {
         self.timestamp = new Date();
-        self.illuminance = Number(data);
+        self.illuminance = Number(data.illuminance);
```

Comment:
I should remove illuminance from simulator and preserve the original line to keep compatibility with bh driver